safehttp/form.go

```diff
@@ -212,9 +213,19 @@ func (f *MultipartForm) File(param string) []*multipart.FileHeader {
 	if !ok {
 		return nil
 	}
+	sanitise(fh)
 	return fh
 }
 
+// sanitise removes trailing path separators from all file names.
+// This is to ensure that uploaded files are not stored outside of the
+// designated directory.
+func sanitise(fhs []*multipart.FileHeader) {
```

Comment:
The name is a bit too generic to be in the safehttp package.

Reply:
Channged to sanitiseFilenames